src/Playwright.Tests/PageNetworkResponseTests.cs

```diff
@@ -63,7 +64,9 @@ public async Task ShouldReturnJson()
         public async Task ShouldWorkWithGenerics()
         {
             var response = await Page.GotoAsync(TestConstants.ServerUrl + "/simple.json");
-            Assert.Equal("bar", (await response.JsonAsync<TestClass>()).Foo);
+            JsonDocument doc = await response.JsonAsync();
```

Comment:
The generic JsonAsync would be way more useful than this. I don't think people would use this with all these required boilerplate.

Reply:
> The generic JsonAsync would be way more useful than this. 

I don't this force-converting to camel case is going to have a single use, default one can at least be used...